apps/src/lib/ui/lti/sync/types.ts

```diff
@@ -0,0 +1,26 @@
+interface LtiSection {
+  name: string;
+  size: number;
+}
+
+export type LtiSectionMap = {[sectionId: string]: LtiSection};
+
+export interface LtiSectionSyncResult {
+  all?: LtiSectionMap;
+  updated?: LtiSectionMap;
+  error?: string;
+}
+
+export interface LtiSectionSyncDialogProps {
```

Comment:
With the props defined here, do we still need `LtiSectionSyncDialog.propTypes` in `LtiSectionSyncDialog.tsx`?